docs/meshstack.identifiers.md

```diff
@@ -0,0 +1,26 @@
+---
+id: meshstack.identifiers
+title: Identifiers
+---
+
+Objects in meshcloud can have three kinds of properties used to refer to them.
```

Comment:
here its meshcloud directly below it is meshStack. I would proposed to go with meshStack